src/ralph/domains/models/domains.py

```diff
@@ -72,7 +72,19 @@ class DNSProvider(
     pass
 
 
-class Domain(AdminAbsoluteUrlMixin, BaseObject):
+class AdditionalService(
```

Comment:
what is `AdditionalService`? Could you name it something less-generic? maybe `RegistrantAdditionalServices` or `DomainProviderAdditionalServices`?

Reply:
I'll change it to DomainProviderAdditionalServices